api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/NullRPC.java

```diff
@@ -0,0 +1,14 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
```

Comment:
Why this class is not in components package together with NullProcedure and NullResourceAPI?

Reply:
You are right, I will move it.